### PR TITLE
A little bit of scripts cleanup

### PR DESCRIPTION
**Pull request by drAlberT**
- Dockerfile: replace tab with spaces to have "universal" alignment
- shell scripts: some best practices (mainly `shellcheck` suggested)

### CODE REVIEW

**Comment by JasonRivers** [comment]
Thanks for your contribution, I will review this in the next couple of days, the changes seem sane, other than I merged your other PR first which has caused a conflict on this one, unless you wish to pull the change in to this PR to fix the conflict.

J

**Comment by JasonRivers** [comment]
I've also updated the image to nagioscore 4.3.4, you may wish to pull those changes in, too

**Comment by drAlberT** [comment]
ok great .. I'm going to rebase right now (I was working on updating to 4.3.4 too eheh )

**Comment by drAlberT** [comment]
It should be ok now ;)

File: start.sh [hunk 1]
```diff
@@ -14,33 +14,33 @@ if [ -z "$(ls -A /opt/nagios/var)" ]; then
...
-  htpasswd -c -b -s ${NAGIOS_HOME}/etc/htpasswd.users ${NAGIOSADMIN_USER} ${NAGIOSADMIN_PASS}
-  chown -R nagios.nagios ${NAGIOS_HOME}/etc/htpasswd.users
+if [ ! -f "${NAGIOS_HOME}/etc/htpasswd.users" ] ; then
+  htpasswd -c -b -s "${NAGIOS_HOME}/etc/htpasswd.users" "${NAGIOSADMIN_USER}" "${NAGIOSADMIN_PASS}"
+  chown -R nagios.nagios "${NAGIOS_HOME}/etc/htpasswd.users"
 fi
 
 shutdown() {
   echo Shutting Down
   ls /etc/service | SHELL=/bin/sh parallel --no-notice sv force-stop {}
```
Comment (drAlberT) [inline start.sh]: @JasonRivers can you explain me this line? .. I don't get the point of the `ls` ..

thanks

Comment (JasonRivers) [reply start.sh]: This was added in my @shane-axiom IIRC what this does is list the directories in /etc/service  and issues a stop on each of the services for that directory.
It's better than a `for SERVICE in $(ls /etc/service)` as it allows us to run the stop in parallel

Comment (JasonRivers) [reply start.sh]: See commit b768c55 for what @shane-axiom added to handle signals

Comment (drAlberT) [reply start.sh]: @JasonRivers I was confused by the name `/etc/service` .. I supposed it is a file (being singular)
 .. maybe changing its name to `/etc/services.d/` could be more explicit ;)

Thanks!

**Pull request merged**